src/workbench/queryWorkbench.ts

```diff
@@ -43,9 +50,12 @@ export class QueryWorkbench {
         if (activeTextEditor && activeTextEditor.document.languageId === "SQL++") {
             // Get the text content of the active text editor.
             const query = activeTextEditor.document.getText();
+            const queryContext = this.editorToContext.get(activeTextEditor.document.uri.toString());
+            const queryContextString = queryContext && (queryContext?.bucketName + '.' + queryContext?.scopeName);
```

Comment:
Use String Interpolation instead of concatenation. If it is done somewhere else please apply the same

Reply:
Used string interpolation in every concatenated string